### PR TITLE
compoz.ts : add undo action shortcut

### PR DESCRIPTION
**Pull request by miraheka2305**

### CODE REVIEW

**Review by shuLhan — APPROVED** [review]

File: src/compoz.ts [hunk 1]
```diff
@@ -317,13 +317,16 @@ export class Compoz {
...
 
-		this.elInput.onkeyup = () => {
+		this.elInput.onkeyup = e => {
 			const contentHTML = this.getContentHTML()
 			if (contentHTML === "") {
 				this.disableButtonSend()
 			} else {
 				this.enableButtonSend()
 			}
+			if (e.keyCode === 90) {
```
Comment (shuLhan) [inline src/compoz.ts]: What is 90?

Comment (miraheka2305) [reply src/compoz.ts]: Its the keyCode for (ctrl + Z)

Comment (shuLhan) [reply src/compoz.ts]: We should add it as comment or as constant, instead of number.

**Review by shuLhan — CHANGES_REQUESTED** [review]

**Comment by miraheka2305** [comment]
I've changed it as a constant
Please check again @shuLhan 
Thankyou

**Review by shuLhan — APPROVED** [review]

**Pull request merged**